gems/pending/util/extensions/miq-string.rb

```diff
@@ -8,147 +8,6 @@ def miqEncode
-  #
-  #   "active_record".camelize                # => "ActiveRecord"
-  #   "active_record".camelize(:lower)        # => "activeRecord"
-  #   "active_record/errors".camelize         # => "ActiveRecord::Errors"
-  #   "active_record/errors".camelize(:lower) # => "activeRecord::Errors"
-  def camelize(first_letter = :upper)
-    case first_letter
-    when :upper then ActiveSupport::Inflector.camelize(self, true)
-    when :lower then ActiveSupport::Inflector.camelize(self, false)
-    end
-  end unless method_defined?(:camelize)
-
-  # Capitalizes all the words and replaces some characters in the string to create
-  # a nicer looking title. +titleize+ is meant for creating pretty output. It is not
-  # used in the Rails internals.
-  #
-  # +titleize+ is also aliased as +titlecase+.
-  #
-  #   "man from the boondocks".titleize # => "Man From The Boondocks"
-  #   "x-men: the last stand".titleize  # => "X Men: The Last Stand"
-  def titleize
-    ActiveSupport::Inflector.titleize(self)
-  end unless method_defined?(:titleize)
-
-  # The reverse of +camelize+. Makes an underscored, lowercase form from the expression in the string.
-  #
-  # +underscore+ will also change '::' to '/' to convert namespaces to paths.
-  #
-  #   "ActiveRecord".underscore         # => "active_record"
-  #   "ActiveRecord::Errors".underscore # => active_record/errors
-  def underscore
-    ActiveSupport::Inflector.underscore(self)
-  end unless method_defined?(:underscore)
-
-  # Replaces underscores with dashes in the string.
-  #
-  #   "puni_puni" # => "puni-puni"
-  def dasherize
-    ActiveSupport::Inflector.dasherize(self)
-  end unless method_defined?(:dasherize)
-
-  # Removes the module part from the constant expression in the string.
-  #
-  #   "ActiveRecord::CoreExtensions::String::Inflections".demodulize # => "Inflections"
-  #   "Inflections".demodulize                                       # => "Inflections"
-  def demodulize
-    ActiveSupport::Inflector.demodulize(self)
-  end unless method_defined?(:demodulize)
-
-  # Creates the name of a table like Rails does for models to table names. This method
-  # uses the +pluralize+ method on the last word in the string.
-  #
-  #   "RawScaledScorer".tableize # => "raw_scaled_scorers"
-  #   "egg_and_ham".tableize     # => "egg_and_hams"
-  #   "fancyCategory".tableize   # => "fancy_categories"
-  def tableize
-    ActiveSupport::Inflector.tableize(self)
-  end unless method_defined?(:tableize)
-
-  # Create a class name from a plural table name like Rails does for table names to models.
-  # Note that this returns a string and not a class. (To convert to an actual class
-  # follow +classify+ with +constantize+.)
-  #
-  #   "egg_and_hams".classify # => "EggAndHam"
-  #   "posts".classify        # => "Post"
-  #
-  # Singular names are not handled correctly.
-  #
-  #   "business".classify # => "Busines"
-  def classify
-    ActiveSupport::Inflector.classify(self)
-  end unless method_defined?(:classify)
```

Comment:
> @Fryguy This file itself requires 'active_support/inflector'

@bdunne Can you expand these conditionals and run the MiqString spec outside of rails to see if these methods are already defined by the require of inflector?